SetupScript.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+if [ $EUID -ne 0 ]; then
+	echo "Please run as root user"
+	exit 1
+fi
+#base script to fresh install systems onto base image for raspberry pi
+
+#check internet connection
+ping -q -c1 google.com > /dev/null
+
+if [ $? -ne 0 ]; then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#configure displays
+git clone https://github.com/UCSolarCarTeam/Epsilon-Raspberry.git /opt/Epsilon-Raspberry/
+cp /opt/Epsilon-Raspberry/primary/xorg.conf /etc/X11/ 
+cp /opt/Epsilon-Raspberry/primary/config.txt /boot/
+
+#install rabbit-mq
+echo 'deb http://www.rabbitmq.com/debian/ testing main' | tee /etc/apt/sources.list.d/rabbitmq.list && apt-get update && apt-get install rabbitmq-server
```

Comment:
Let's move the `apt-get update && apt-get install rabbitmq-server` commands to separate commands.